### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Yilikal250**
Added the contact form with Santiago who was my coding partner we added a form.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
-# Template2Portfolio
-Personal portifolio
+# Project Name:-
```
Comment (TedLivist) [inline README.md:1]: - [ ] In your README, kindly include detailed instructions on how users can set up and run your app locally.

File: index.html [hunk 2]
```diff
@@ -55,5 +56,265 @@ <h2>I'm a Software Developer</h2>
...
+          <textarea
+            name="textarea"
+            id="textarea"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <div>
+            <button class="footer-button" type="submit">Get in touch</button>
+          </div>
```
Comment (TedLivist) [inline index.html:292]: - [ ] Kindly ensure that the effects for hover, active (pressed), and disabled are implemented for your button as we have in the template.

  buttons-color

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Yilikal250,

Good job so far!
✔️ Nice design
✔️ Form fields are required
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**